integrations/mysql/documentation.yaml

```diff
@@ -0,0 +1,99 @@
+exporter_type: sidecar
+app_name_short: MySQL
+app_name: {{app_name_short}}
+app_site_name: MySQL
+app_site_url: https://www.mysql.com/
+exporter_name: the Mysql exporter
+exporter_pkg_name: mysqld_exporter
+exporter_repo_url: https://github.com/prometheus/mysqld_exporter
+dashboard_available: true
+minimum_exporter_version: v0.14.0
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    name: mysql
+  spec:
+    serviceName: mysql
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: mysql
+    template:
+      metadata:
+        labels:
+  +       app.kubernetes.io/name: mysql
+      spec:
+        containers:
+  +     - name: exporter
+  +       image: prom/mysqld-exporter:v0.14.0
+  +       env:
+  +         - name: DATA_SOURCE_NAME
+  +           value: root:password@(localhost:3306)/
+  +       ports:
+  +       - containerPort: 9104
+  +         name: prometheus
+        - image: mysql:5.7
+          name: mysql
+          env:
+          - name: MYSQL_ROOT_PASSWORD
+            value: password
+          - name: MYSQL_USER
+            value: sbtest
+          - name: MYSQL_PASSWORD
+            value: password
+          - name: MYSQL_DATABASE
+            value: sbtest
+          ports:
+          - containerPort: 3306
+            name: mysql
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: mysql
+    labels:
+      app.kubernetes.io/name: mysql
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: prometheus
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: mysql
+additional_prereq_info: |
+  See the exporter [Required Grants](https://github.com/prometheus/mysqld_exporter#required-grants){:class=external}
```

Comment:
For information about creating a least-privileged user, see the exporter [Required Grants](https://github.com/prometheus/mysqld_exporter#required-grants){:class=external} documentation.


(We're supposed to tell users "why" before "what".)